test/reservation_desk_test.rb

```diff
@@ -0,0 +1,317 @@
+require_relative "test_helper"
+
+describe "ReservationDesk class" do
+  before do
+    @reservation_desk = Hotel::ReservationDesk.new()
+  end
+
+  describe "ReservationDesk instantiation" do
+    it "is an instance of ReservationDesk" do
+      expect(@reservation_desk).must_be_kind_of Hotel::ReservationDesk
+    end
+  end
+
+  describe "rooms" do
+    it "returns an array" do
```

Comment:
these two it blocks  (`returns and array` and `returns an array of Rooms`) could be included in a single it block